packages/medusa/src/utils/calculate-price-tax-amount.ts

```diff
@@ -0,0 +1,29 @@
+import { featureFlagRouter } from "../loaders/feature-flags"
```

Comment:
thought: I think we would benefit from making the util pure by not importing the feature flag, but instead just making `includesTax` handle the calculation regardless of what feature flags are enabled. The responsibility to call it correctly will then fall onto the callee. Wdyt?

Reply:
I know that this is from a previous PR, but just didn't think of it before now :D

Reply:
I totally agree, the flag should be part of the condition passed to the includesTax options to the util ^^ i ll make a pr tomorrow to change it where it is used so that it does not impact the dependent pr’s 😀 is that ok?